src/explore-education-statistics-admin/src/services/preReleaseUserService.ts

```diff
@@ -4,13 +4,27 @@ export interface PreReleaseUser {
   email: string;
 }
 
+export interface PreReleaseInvitePlan {
+  alreadyAccepted?: string[];
+  alreadyInvited?: string[];
+  invitable?: string[];
```

Comment:
Are these supposed to be optional? Seems like the backend API always returns empty lists (which would be the right thing to do)? 

If these aren't optional, then I would also adjust the confirmation modal checks as we could simplify checks like:
`invitePlan.alreadyInvited && invitePlan.alreadyInvited.length > 0` to simply ` invitePlan.alreadyInvited.length > 0`